### DISCUSSION TITLE
`about` command does not show variables with `null` value

### DISCUSSION THREAD

**Discussion by siarheipashkevich** [laravel/framework#44517] [Q&A]
Is it ok, that the `php artisan about` command does not show variables with `null` value?

For example, I set in `.env` file `CACHE_DRIVER=null` and this variable is not displayed:
![image](https://user-images.githubusercontent.com/7753600/194703059-e5af9730-a8f2-4223-b8b1-e698128c8f03.png)

But If I set to another value then it will be displayed:
![image](https://user-images.githubusercontent.com/7753600/194703085-22fcc040-a179-4e43-b5c3-0b0824febe01.png)

@jbrooksuk @nunomaduro

**Comment by jbrooksuk** [comment]
They're filtered out since the drivers wouldn't be enabled.

If this is unexpected, we could show them.

**Reply by siarheipashkevich** [reply]
I think will be better to show all possible drivers and the values for them.

What do you think?

**Reply by henzeb** [reply]
Or at least a flag that unhides them
